docs/docs/faq.md

```diff
@@ -0,0 +1,5 @@
+---
+title: FAQ
+---
+## Why am I still seeing vulnerable images?
+Eraser currently targets **non-running** images, so any vulnerable images that are currently running will not be removed. In addition, The default vulnerability scanning with Trivy removes images with `CRITICAL` vulnerabilities. Any images with lower vulnerabilities will not be removed. This can be configured with the `--severity` flag.
```

Comment:
```suggestion
Eraser currently targets **non-running** images, so any vulnerable images that are currently running will not be removed. In addition, the default vulnerability scanning with Trivy removes images with `CRITICAL` vulnerabilities. Any images with lower vulnerabilities will not be removed. This can be configured with the `--severity` flag.
```